src/firebase/auth/logout.ts

```diff
@@ -0,0 +1,13 @@
+import firebase_app from '@/firebase/config';
```

Comment:
make this function be executed on the server by including the "use server" directive at the very top!